plugin/pick.vim

```diff
@@ -56,7 +60,7 @@ function! s:FileListCommand()
 endfunction
 
 function! s:GitListCommand(file)
-  return "git ls-files " . a:file . " --cached --exclude-standard --others"
```

Comment:
Reading your commit message, I see your use case. Working in new codebases where I create a new files more often than in I would in a legacy codebases is why I want to keep the existing behavior.

What do you think of making `GitListCommand` public, and maybe renaming it to `PickGitListCommand` to make it overridable by the user?